### ISSUE TITLE
Default attribute declared as list instead of hash

### ISSUE THREAD

**Issue by RulerOf** [Numergy/cloud-init-cookbook#2]
https://github.com/Numergy/cloud-init-cookbook/blob/4b0f3d9dca11f7cc4ccf22aad3e5cc51458dfa17/attributes/default.rb#L19

Linked above, the `default['cloud_init']['cfgs']` in the cookbook attributes is set to `[]`, which causes merge errors in Chef 12.  Setting this:

```ruby
default['cloud_init']['cfgs']['bq_core']['config']['disable_root'] = false
```

Yields this error:
```
       ================================================================================
       Recipe Compile Error in /tmp/kitchen/cache/cookbooks/bq-core/attributes/cloud.rb
       ================================================================================
       
       TypeError
       ---------
       no implicit conversion of String into Integer
       
       Cookbook Trace:
       ---------------
         /tmp/kitchen/cache/cookbooks/bq-core/attributes/cloud.rb:2:in `from_file'
       
       Relevant File Content:
       ----------------------
       /tmp/kitchen/cache/cookbooks/bq-core/attributes/cloud.rb:
       
         1:  # Cloud-init configs
         2>> default['cloud_init']['cfgs']['bq_core']['config']['disable_root'] = false
         3:  
```

I can override it by unfurling the hash or using an attribute of higher precedence for now.

**Comment by Sliim** [comment]
Hi! Thanks for your report, it's a typo, I will fix that

**Comment by Sliim** [comment]
Fixed in cloud_init `0.2.1`. Thx!
